rpc/server_auth.go

```diff
@@ -374,7 +374,8 @@ func (ss *simpleServer) ensureAuthed(ctx context.Context) (context.Context, erro
 				"error", err,
 				"empty expected audience", len(ss.authAudience) == 0,
 				"expected audience list", audienceList)
-			//return nil, status.Error(codes.Unauthenticated, "invalid audience")
+			// TODO: re-enable once auth migration is complete
+			// return nil, status.Error(codes.Unauthenticated, "invalid audience")
```

Comment:
flyby: had to add a space after `//` to avoid a linter crash/deletion. added a TODO to clarify why we left this code in as a comment.

Reply:
ignore this - the issue was resolved in `main`